<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="360LP048" name="com.edureka.community.VerifyOnlyEnrolledCourseIsShowInSelectedCourseDropdown" tests="3" failures="1" timestamp="8 Sep 2016 09:28:11 GMT" time="442.652" errors="2">
  <testcase name="test_007VerifyErrorMessageForBlankField" time="0.000" classname="com.edureka.community.VerifyOnlyEnrolledCourseIsShowInSelectedCourseDropdown">
    <error type="java.lang.Throwable" message="Method VerifyOnlyEnrolledCourseIsShowInSelectedCourseDropdown.test_007VerifyErrorMessageForBlankField()[pri:0, instance:com.edureka.community.VerifyOnlyEnrolledCourseIsShowInSelectedCourseDropdown@e45f292] depends on not successfully finished methods">
      <![CDATA[java.lang.Throwable: Method VerifyOnlyEnrolledCourseIsShowInSelectedCourseDropdown.test_007VerifyErrorMessageForBlankField()[pri:0, instance:com.edureka.community.VerifyOnlyEnrolledCourseIsShowInSelectedCourseDropdown@e45f292] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1040)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.TestNG.privateMain(TestNG.java:1325)
	at org.testng.TestNG.main(TestNG.java:1294)
]]>
    </error>
  </testcase> <!-- test_007VerifyErrorMessageForBlankField -->
  <testcase name="test_004VerifyQuestionTitle" time="0.000" classname="com.edureka.community.VerifyOnlyEnrolledCourseIsShowInSelectedCourseDropdown">
    <error type="java.lang.Throwable" message="Method VerifyOnlyEnrolledCourseIsShowInSelectedCourseDropdown.test_004VerifyQuestionTitle()[pri:0, instance:com.edureka.community.VerifyOnlyEnrolledCourseIsShowInSelectedCourseDropdown@e45f292] depends on not successfully finished methods">
      <![CDATA[java.lang.Throwable: Method VerifyOnlyEnrolledCourseIsShowInSelectedCourseDropdown.test_004VerifyQuestionTitle()[pri:0, instance:com.edureka.community.VerifyOnlyEnrolledCourseIsShowInSelectedCourseDropdown@e45f292] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1040)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.TestNG.privateMain(TestNG.java:1325)
	at org.testng.TestNG.main(TestNG.java:1294)
]]>
    </error>
  </testcase> <!-- test_004VerifyQuestionTitle -->
  <testcase name="test_003VerifyOnlyEnrolledCourseIsShowInSelectedCourseDropdown" time="442.652" classname="com.edureka.community.VerifyOnlyEnrolledCourseIsShowInSelectedCourseDropdown">
    <failure type="java.lang.AssertionError" message="Element Locator ://input[@id=&#039;input&#039;] Not found expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Element Locator ://input[@id='input'] Not found expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.edureka.util.DriverHelper.getAttribute(DriverHelper.java:433)
	at com.edureka.pages.ForumsPage.verifyPage(ForumsPage.java:26)
	at com.edureka.community.VerifyOnlyEnrolledCourseIsShowInSelectedCourseDropdown.test_003VerifyOnlyEnrolledCourseIsShowInSelectedCourseDropdown(VerifyOnlyEnrolledCourseIsShowInSelectedCourseDropdown.java:136)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.TestNG.privateMain(TestNG.java:1325)
	at org.testng.TestNG.main(TestNG.java:1294)
]]>
    </failure>
  </testcase> <!-- test_003VerifyOnlyEnrolledCourseIsShowInSelectedCourseDropdown -->
</testsuite> <!-- com.edureka.community.VerifyOnlyEnrolledCourseIsShowInSelectedCourseDropdown -->
